systemvm/debian/root/health_checks/dhcp_check.py

```diff
@@ -41,11 +43,15 @@ def main():
                 break
 
         if not foundEntry:
-            print "Missing elements in dhcphosts.txt - " + entry
-            exit(1)
-
-    print "All " + str(len(vMs)) + " VMs are present in dhcphosts.txt"
-    exit(0)
+            failedCheck = True
+            failureMessage = failureMessage + entry + "\n"
```

Comment:
this is inside a loop. if we detect multiple failures we just log the last one?

Reply:
We are appending all failures Daan. The intention is that we want to return all missing entries so instead of returning on first failure we want to capture all.

Reply:
ok, me blind. I would expect += for that sorry, my bad